packages/documentation/src/content/docs/playground/autopeering.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Autopeering
+---
+
+If you want to start one local instance of Rafiki and peer it automatically to [Rafiki.money](https://rafiki.money), you can run the following commands:
+
+```
+pnpm localenv:compose:autopeer
+
+// OR to start with Postgres db
+pnpm localenv:compose:psql:autopeer
+```
+
+Your local Rafiki instance will be peered automatically with the remote [Rafiki.money](https://rafiki.money) instance.
+The required services will be exposed externally using [tunnelmole](https://www.npmjs.com/package/tunnelmole) package.
+The exposed ports are 3000(open-payments), 3006(auth server), 3002(ILP connector).
+
+To use the [Postman API examples](../overview.md#postman--open-payments-apis) follow the steps:
```

Comment:
this one is broken

Reply:
Still? Ok, will fix later. Have the meeting with the hackathon gang now.